swift/internal/compiling.bzl

```
@@ -927,6 +943,18 @@ def _global_module_cache_configurator(prerequisites, args):
             paths.join(prerequisites.bin_dir.path, "_swift_module_cache"),
         )
 
+def _tmpdir_module_cache_configurator(prerequisites, args):
+    """Adds flags to enable a stable tmp directory module cache."""
+
+    args.add(
+        "-module-cache-path",
+        paths.join(
+            "/private/tmp/__build_bazel_rules_swift",
```

```suggestion
            "/tmp/__build_bazel_rules_swift",
```

if there's no difference I think using this would be preferred since I view that symlink as a bit of an implementation detail, but maybe there is?